lib/matplotlib/text.py

```diff
@@ -2124,25 +2132,32 @@ def __init__(self, s, xy,
             ?                any key for :class:`matplotlib.patches.PathPatch`
             ===============  ==================================================
 
-            Defaults to None
+            Defaults to None, i.e. no arrow is drawn.
 
-        annotation_clip : bool, optional
-            Controls the visibility of the annotation when it goes
+        annotation_clip : bool or None, optional
+            Whether to draw the annotation when the annotation point *xy* is
             outside the axes area.
 
-            If `True`, the annotation will only be drawn when the
-            ``xy`` is inside the axes. If `False`, the annotation will
-            always be drawn regardless of its position.
+            - If *True*, the annotation will only be drawn when *xy* is
+              within the axes.
+            - If *False*, the annotation will always be drawn.
+            - If *None*, the annotation will only be drawn when *xy* is
+              within the axes and *xycoords* is 'data'.
+
+            Defaults to *None*.
 
-            The default is `None`, which behave as `True` only if
-            *xycoords* is "data".
+        **kwargs
+            Additional kwargs are passed to `~matplotlib.text.Text`.
 
         Returns
         -------
-        Annotation
+        `.Annotation`
```

Comment:
This renders as 
![image](https://user-images.githubusercontent.com/23121882/45922529-4847b700-bece-11e8-8f76-a4bb6e75b808.png)